corehq/apps/users/models.py

```diff
@@ -1599,11 +1599,10 @@ def location_group_ids(self):
         location_type = self.location.location_type_object
         if location_type.shares_cases:
             if location_type.view_descendants:
-                from corehq.apps.locations.models import SQLLocation
-                sql_loc = SQLLocation.objects.get(location_id=self.location._id)
                 return [
                     loc.case_sharing_group_object(self._id)._id
-                    for loc in sql_loc.get_descendants()
+                    for loc in self.sql_location.get_descendants(include_self=True)
+                    if loc.location_type.shares_cases
```

Comment:
second issue - descendants that didn't have `shares_cases=True` were still showing up